### DISCUSSION TITLE
Ability to Customize Login Page Logo, Site Icon and System Name when using Multi-Site

### DISCUSSION THREAD

**Discussion by bgarrant** [craftcms/cms#8606] [Ideas]
### Description

It would be so much better for site editors if we could customize the Login Page Logo, Site Icon and System Name per site when using multi-site configuration.  Have it change based on the site selection or user permissions. Maybe an environment or config variable.

It would also be nice to be able to change the site using the left top menu: https://cl.ly/a09e510075fe.

So if a user only has access to Site B, when they login they would be presented with the default Login Page Logo, Site Icon and System Name if they have access to both, or a specific Login Page Logo, Site Icon and System Name if they only have access to one site.

The login page I know is tied to one URL for licensing, and that can stay as it is, or even better list the two System Names on the page possibly if no logo is chosen?  Just a thought but this is minor part of the request.

### Additional info

- Craft version: 3.0.30.1
- PHP version: 7.1

**Comment by brandonkelly** [comment]
Currently there is no CP-wide selected site, but we may move the CP in that direction in Craft 4.
